examples/direct_fidelity_estimation.py

```diff
@@ -109,7 +109,7 @@ def _estimate_pauli_traces_clifford(n_qubits: int,
         n_qubits: An integer that is the number of qubits.
         clifford_state: The basis of the Pauli states with non-zero probability.
         n_clifford_trials: An integer that is the number of Pauli states to
-            sample.
+            sample. If set to 0, we compute the exhaustive list of Pauli states.
```

Comment:
Zero seems like the wrong value for this. I'd expect zero to return the maximally uninformed prior instead of doing exhaustive search.

Maybe what we should do instead is use a completely different value, like `float("inf")` or `"all"` or even a separate argument `exhaustive=True` that can't be specified simultaneously. Or perhaps this should even be a completely different method. This seems like a rather large change in behavior.

Reply:
@tonybruguier-google Why not just use `None`?  This is the usual way in Python to indicate the value isn't specified.